### ISSUE TITLE
Incremental AutoSave?

### ISSUE THREAD

**Issue by GOvEy1nw** [bonjorno7/PowerSave#18]
Hey, is it possible to incrementally save the autosaves? instead of overwriting themselves? I assumed it'd be a simple case of setting autosave to 'Custom' and then using a date format or something at the end of the file name, but I can't seem to get it to work. Is there's something I can add next to {name}, such as {name}_{date}, or something to autosave in increments, but limit the max number of autosaves?

Essentially, I want to be able to have say 10 autosave increments saved in //AutoSave/ folder next to the original file, and they overwrite from the first one again once it's reached the limit of 10.

**Comment by 2ndHouse** [comment]
Yeah I was hoping to see this functionality in PowerSave as well. Some options for setting a certain number of autosaves. Any chance of seeing that in a future version?

To the OP...I know your request is a couple of years old, but if you are still looking for that functionality, this add-on does what you are looking for:
https://extensions.blender.org/add-ons/incremental-auto-save/
